src/olympia/addons/serializers.py

```diff
@@ -1342,8 +1343,12 @@ def fake_version_object(self, obj, data, channel):
             for app_id, compat_dict in data.get('compatible_apps', {}).items():
                 app_name = APPS_ALL[int(app_id)]
                 compatible_apps[app_name] = ApplicationsVersions(
-                    min=AppVersion(version=compat_dict.get('min_human', '')),
-                    max=AppVersion(version=compat_dict.get('max_human', '')),
+                    min=AppVersion(
+                        created=None, version=compat_dict.get('min_human', '')
+                    ),
+                    max=AppVersion(
+                        created=None, version=compat_dict.get('max_human', '')
+                    ),
```

Comment:
re-use the file creation date like we do with `Version` above?

Reply:
But I guess if we don't expose any of these dates in the serializer respones it's redundant anyway...

Reply:
For `AppVersion` that'd be wrong IMHO, because they are not tied to the `Version` at all - `AppVersion`s are created by us when a new Firefox versions is created... 

I wanted to do it for `ApplicationsVersions`, but it turns out this model doesn't inherit from `ModelBase` and therefore doesn't have a `created` date (https://github.com/mozilla/addons-server/issues/20013#issuecomment-1328862068).